tests/integration/targets/vmware_ops_vcenter_host_connection_test/tasks/main.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Test vCenter host connection role
+  block:
+    - name: Add Resource Pool
+      ansible.builtin.include_tasks: manage_resource_pool_role.yml
+      vars:
+        required_state: present
+      tags: eco-vcenter-ci
+
+    - name: Provision Virtual ESXI and set ESXI hostname
+      ansible.builtin.include_tasks: provision_esxi.yml
+      tags: eco-vcenter-ci
+
+    - name: Add ESXI Host to vCenter Cluster
+      ansible.builtin.import_role:
+        name: cloud.vmware_ops.vcenter_host_connection
+      vars:
+        vcenter_host_connection_state: present
+
+    - name: Gather ESXI Host facts
+      community.vmware.vmware_host_facts:
+        hostname: "{{ vcenter_hostname }}"
+        username: "{{ vcenter_username }}"
+        password: "{{ vcenter_password }}"
+        validate_certs: false
+        esxi_hostname: "{{ vcenter_host_connection_esxi_hostname }}"
+      register: _host_facts
+
+    - name: Verify ESXI Host connected to vCenter Cluster
+      ansible.builtin.assert:
+        that:
+          - _host_facts.ansible_facts.cluster == cluster_name
+          - _host_facts.ansible_facts.ansible_host_connection_state == 'connected'
+
+    - name: Remove ESXI Host from vCenter Cluster
+      ansible.builtin.import_role:
+        name: cloud.vmware_ops.vcenter_host_connection
+      vars:
+        vcenter_host_connection_state: absent
```

Comment:
Can we add a check also after the removal of the esxi host?

Reply:
Sure. I thought I added it